### ISSUE TITLE
AWS.Tools: AWS common parameters descriptions are missing from help

### ISSUE THREAD

**Issue by juneb** [aws/aws-tools-for-powershell#138]
When you run Get-Help -Full, Get-Help -Parameter {*, param_name}, the descriptions of some of the AWS common parameters are missing. 

This issue is limited to the AWS.Tools modules. It does not affect AWSPowerShell or AWSPowerShell.NetCore. It is not related to the installation method. It occurs when you use Install-Module or Install-AWSToolModule. 

The descriptions of the following parameters are missing:

```
-AccessKey <System.String>
    Amazon.PowerShell.Common.AWSCommonArgumentsCmdlet.AccessKey

-Credential <Amazon.Runtime.AWSCredentials>
    Amazon.PowerShell.Common.AWSCommonArgumentsCmdlet.Credential

-EndpointUrl <System.String>
    Amazon.PowerShell.Common.ServiceCmdlet.EndpointUrl

-NetworkCredential <System.Management.Automation.PSCredential>
    Amazon.PowerShell.Common.AWSCommonArgumentsCmdlet.NetworkCredential

-ProfileLocation <System.String>
    Amazon.PowerShell.Common.AWSCommonArgumentsCmdlet.ProfileLocation

-ProfileName <System.String>
    Amazon.PowerShell.Common.AWSCommonArgumentsCmdlet.ProfileName

-Region <System.Object>
    Amazon.PowerShell.Common.AWSCommonArgumentsCmdlet.Region

-SecretKey <System.String>
    Amazon.PowerShell.Common.AWSCommonArgumentsCmdlet.SecretKey

-SessionToken <System.String>
    Amazon.PowerShell.Common.AWSCommonArgumentsCmdlet.SessionToken
```

There is a help file for each module, e.g. <ModuleName>.dll-help.xml, but I'm not sure how you do the common parameters (completers?). I'm sure you'll figure this out before I can take apart the module and figure out how it works. 

**Expected Behavior**

If you're describing a bug, tell us what should happen. If you're suggesting a change/improvement, tell us how it should work.

**Current Behavior**

If describing a bug, tell us what happens instead of the expected behavior.
Include full errors, uncaught exceptions, stack traces, and relevant logs.
If service responses are relevant, please include Fiddler-captured wirelogs.
If suggesting a change/improvement, explain the difference from current behavior.

**Possible Solution**

Not required, but suggest a fix/reason for the bug, or ideas how to implement the addition or change.

**Steps to Reproduce (for bugs)**

Provide a self-contained, concise snippet of code that can be inserted into a PowerShell session to replicate the bug. Including business logic or unrelated code makes diagnosis more difficult.

**Context**

How has this issue affected you? What are you trying to accomplish? Providing context helps us come up with a solution that is most useful in the real world.

**Your Environment**

Include as many relevant details about the environment where the bug was discovered.
* Affected module (_AWSPowerShell_, _AWSPowerShell.NetCore_ or _AWS.Tools.*_): AWS.Tools
* Module version as reported by `Get-AWSPowerShellVersion`:  4.0.5.0
* Operating System and version as reported by `$PSVersionTable.OS`: 
* PowerShell version as reported by `$PSVersionTable.PSEdition`:

PS C:\ps-test> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      7.0.0-rc.2
PSEdition                      Core
GitCommitId                    7.0.0-rc.2
OS                             Microsoft Windows 10.0.17763
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0

**Comment by matteo-prosperi** [comment]
Thank you for reporting this. We will look into it by the next release.

**Comment by ashishdhingra** [comment]
Fixed in AWS Tools for PowerShell version `4.1.281.0` (you would need to reload PowerShell terminal/prompt for changes to take effect).
